src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -205,3 +206,61 @@ reprocessing all the failed events registered in event dead letters.
 If for some other reason you don't need to redeliver all events, you have more fine-grained operations allowing you to
 [redeliver group events](manage-webadmin.html#Redeliver_group_events) or even just
 [redeliver a single event](manage-webadmin.html#Redeliver_a_single_event).
+
+## Solving cassandra inconsistencies
+
+Many tables in James have close relationships. Some operation requires cassandra writes to those tables, 
```

Comment:
`Cassandra table`. Also speak of denormalisation as a common noSQL pattern.

Reply:
```
Some operation requires cassandra writes to those tables,  if there is any failure upon in the middle of an operation. It could bring an inconsistent state to the data.
```

It can also be a dirty read, it may be a good idea to emphasis on that some operations require multiple table writes.

Reply:
s/writes/to write

Reply:
> s/writes/to write

It has been replaced by another sentence